src/olympia/reviews/views.py

```diff
@@ -166,7 +166,11 @@ def delete(request, addon, review_id):
     review = get_object_or_404(Review.objects, pk=review_id, addon=addon)
     if not user_can_delete_review(request, review):
         raise PermissionDenied
-    review.delete(user_responsible=request.user)
+    # Log deleting moderated reviews
+    if review.editorreview:
+        review.moderator_delete(user=request.user)
```

Comment:
@eviljeff said that the API call to delete a review calls the `delete` function...